sbc_compassion/models/correspondence_template.py

```diff
@@ -98,9 +98,6 @@ class CorrespondenceTemplate(models.Model):
     pattern_y_max = fields.Integer(
         help='Maximum Y position of the area in which to look for the '
              'pattern inside the template (given in pixels)')
-    checkbox_ids = fields.One2many(
-        'correspondence.lang.checkbox', 'template_id',
-        default=lambda self: self._get_default_checkboxes(), copy=True)
```

Comment:
Sorry it was not clear, in fact you must keep the field, but remove the `default` parameter.